exporter/opentelemetry-exporter-otlp-proto-common/src/opentelemetry/exporter/otlp/proto/common/trace_encoder/__init__.py

```diff
@@ -208,11 +181,9 @@ def _encode_trace_state(trace_state: TraceState) -> Optional[str]:
 
 
 def _encode_parent_id(context: Optional[SpanContext]) -> Optional[bytes]:
-    if isinstance(context, SpanContext):
-        encoded_parent_id = _encode_span_id(context.span_id)
-    else:
-        encoded_parent_id = None
-    return encoded_parent_id
+    if context:
+        return _encode_span_id(context.span_id)
+    return None
 
 
 def _encode_attributes(
```

Comment:
Can't we use the one in `common` instead of redefining this function?

Reply:
Yes we could